File: Selenium/Activities/activity1.py
from selenium import webdriver
from selenium.webdriver.firefox.service import service as FirefoxService
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    #Open URL
    driver.get("https://v1.training-support.net/")
    print(driver.title)

    #Print Page Title
    print("Page Title: ", driver.title)

    #Find About Us button and Click
    driver.find_element(By.ID, "about-link").click()

    print("About Us Page title: ", driver.title)

File: Selenium/Activities/Activity1.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
	
	public static void main(String[] args) {
		
		// Open the page and print the title
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net");
		driver.getTitle();
		System.out.println("Home page title: " + driver.getTitle());
		
		//Find "About Us" button on the page
		driver.findElement(By.id("about-link")).click();
		System.out.println("About page title: " + driver.getTitle());
		
		driver.close();
		
	}

}
